### ISSUE TITLE
git-clang-format does not work on MS Windows

### ISSUE THREAD

**Issue by fgrie** [ssciwr/clang-format-wheel#36]
As mentioned in the title, `git-clang-format` does not work on MS Windows.

```
PS > git clang-format
Traceback (most recent call last):
  File "C:\Program Files\Python37\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "C:\Program Files\Python37\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Users\username\directory\.venv\Scripts\git-clang-format.exe\__main__.py", line 7, in <module>
  File "c:\users\username\directory\.venv\lib\site-packages\clang_format\__init__.py", line 20, in git_clang_format
  File "c:\users\username\directory\.venv\lib\site-packages\clang_format\__init__.py", line 8, in _run
  File "C:\Program Files\Python37\lib\subprocess.py", line 323, in call
    with Popen(*popenargs, **kwargs) as p:
  File "C:\Program Files\Python37\lib\subprocess.py", line 775, in __init__
    restore_signals, start_new_session)
  File "C:\Program Files\Python37\lib\subprocess.py", line 1178, in _execute_child
    startupinfo)
OSError: [WinError 193] %1 ist keine zulässige Win32-Anwendung
```

This happens because Windows, unlike Linux/UNIIX, is not able to execute a Python file directly, instead, the Python interpreter has to be explicitely called with the script as argument (`python git-clang-format`).

An attempt to fix this is provided in https://github.com/fgrie/clang-format-wheel/commit/57a4e94b7c051b20efbf31230b9d7e4de413862d

I only tested it on my Windows machine where it solves the problem:

```
PS > git clang-format
no modified files to format
```

Validation Linux/UNIX is still required.

**Comment by dokempf** [comment]
Thanks for reporting the problem! Your fix looks like the correct thing to do. Could you open a PR on this repository?

**Comment by fgrie** [comment]
#37

**Comment by dokempf** [comment]
Fixed by #37, tested by #38 and deployed as part of version 14.0.3 Backports will be coming over the next days. Thanks for reporting and contributing!
